tests/cases/case_providers_care_platform.yaml

```diff
@@ -0,0 +1,441 @@
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.create_event_btn$    
+    - Type: click
+      Strategy: id
+      Id: $PAGE.care_platform_call_portal.category_select$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.teleheath_appointment_option$
+    - Type: send_keys
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.subject_input$
+      Value: Automation test subject
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.date_container$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.today_date_date_picker$
+    - Type: send_keys
+      Strategy: id
+      Id: $PAGE.care_platform_call_portal.start_time_picker$
+      Value: 800a
+    - Type: send_keys
+      Strategy: id
+      Id: $PAGE.care_platform_call_portal.end_time_picker$
+      Value: 900p
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.next_btn$
+      # Filling the Situation fields
+      # urgent patient matter
+    - Type: click
+      Strategy: id
+      Id: $PAGE.care_platform_call_portal.urgent_patient_matter_select$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.urgent_patient_matter_yes_option$
+      # reason for request
+    - Type: click
+      Strategy: id
+      Id: $PAGE.care_platform_call_portal.reason_for_request_select$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.reason_for_request_covid_options$
+      # when did symptoms start      
+    - Type: click
+      Strategy: class
+      Id: $PAGE.care_platform_call_portal.symptoms_start_input_container$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.today_date_date_picker$
+      # Click in the submit button
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.submit_create_event_btn$
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_notifications.event_created_notification$
+      # Navigate back to home 
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform.navigation_home$
+set_missed_call_ticket:
+  Environment: Settings 
+  Roles:
+    - Role: desktopChrome
+      Capabilities:
+        chromeOptions:
+          args:
+            - use-fake-ui-for-media-stream
+            - use-fake-device-for-media-stream
+            - no-sandbox
+            - disable-gpu
+      App: desktop   
+    - Role: localiOS
+      App: NeverAlone
+  Actions:   
+      # Navigate to Participants
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform.navigation_participant$
+      # Open the Call Portal
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.miguel_henriquez_one_td$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.call_portal_btn$
+    - Type: sleep
+      Time: 3
```

Comment:
can we wait for a certain condition instead of using this 3 seconds?

Reply:
yes - solved